manifests/cinder-csi-plugin/csi-secret-cinderplugin.yaml

```diff
@@ -4,6 +4,7 @@
 kind: Secret
 apiVersion: v1
 metadata:
-  name: csi-secret-cinderplugin
```

Comment:
Why bothering renaming here?

Reply:
In order to only manage one secret related to OpenStack and not multiple secrets.